jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
Not sure it does without explicitly configuring ES modules in package json (risky) but we're not using anything dynamic right now so it can just be JSON